style.css

```diff
@@ -1 +1,384 @@
+    height: 30%;
+    width: 90%;
+    overflow: hidden;
+    padding: 3%;
+}
+
+.h1-cakes {
+    font-weight: bolder;
+}
+
+.row1-div, .row2-div {
+   display: flex;
+   flex-direction: row;
+   justify-content: space-evenly;
+}
+
+.first-row-article, .second-row-article {
+    margin: 3%;
+    border: 1px solid black;
+    border-radius: 5%;
+    padding: 2%;
+    background-color: rgb(241, 224, 227);
+    width: 30%;
+}
+
+/* images */
+
+.section2-picture {
+    height: auto;
+    width: 100%;
+    border: 2px solid black;
+    border-radius: 2%;
+    padding: 2px;
+}
+
+.section2-picture:hover {
+    transform: scale(1.5);
+    overflow: hidden;
+}
+
+.cakes div article h6 {
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    font-size: 0.9rem;
+    border-bottom: 1px solid grey;
+    width: auto;
+    overflow: hidden;
+
+}
+
+/* footer nav */
+
+.bottom-list {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border-top: 2px solid rgb(50, 6, 82);
+    margin-top:30px;
+    padding: 15px;
+    list-style-type: none;
+    background-color: rgb(228, 112, 208);
+}
+
+footer nav {
+    border: 2px solid black;
+    background-color: rgb(228, 112, 208);
+    margin: 10px 5px;
+}
+
+
+button:hover {
+    border-style: inset;
+}
+
+.media {
+    display: flex;
+    justify-content: center;
+    border-top: 2px solid rgb(50, 6, 82);
+}
+
+.media button {
+    margin: 10px;
+    background-color: rgb(70, 70, 238);
+    border-top: 2px solid black;
+}
+
+/* responsive to desktop */
+
+@media (min-width: 961px) {
+
+    header {
+        margin-bottom: 10%;
+    }
+
+    .logo {
+        border: 2px solid black;
+        height: 100%;
+        width: auto;
+}
+
+    .menu-bar {
+        display:flex;
+        flex-direction: row;
+        justify-content: space-between;
+        height: 7rem;
+        padding-right: 10%;
+        padding-bottom: 2%;
+    }
+
+    #checkbox, label {
+        display: none;
+    }
+
+    .menu {
+        display: flex;
+        justify-content: flex-start;
+        position: absolute;
+        right: 10%;
+        top: 10%;
+        width: 100%;
+        margin-left: 20%;
+        padding: 5%;
+    }
+
+    .menu li {
+        background-color:   rgb(248, 179, 237);
+        border: 2px solid black;
+        text-align: center;
+        font-weight: bolder;
+        margin-left: 5%;
+        margin-right: 5%;
+        padding: 1%;
+    }
+
+    .nav-item {
+        width: 80%;
+        margin-left: 3rem;
+        margin-top: 5rem;
+    }
+
+    #contact {
+        top: 5%;
+        height: 8%;
+        width:5%;
+        border-radius: 40%;
+        background-color: rgb(199, 10, 199);
+    }
+    
+    .hello-text h1 {
+        padding-top: 10%;
+    }
+
+    .carousel {
+       width: 80%;
+       height: auto;
+       border: 2px solid black;
+    }
+
+    .carousel-image {
+        height: 80%;
+        width: 90%;
```

Comment:
You're setting a width AND a height here. This will distort your image. Set the width only, and leave the height to `auto`